src/utils/routingConstants.js

```diff
@@ -3,6 +3,7 @@
 
 export const ROUTE_ABOUT = '/about'
 export const ROUTE_DEFINITIONS = '/definitions'
+export const ROUTE_DETAIL = '/detail'
```

Comment:
Thought we were going to do `/definitions/npm/npmjs/-/foo/1.0` for the route here? need a bit of routing work but that keeps it simple for users. /definitions gets them all the defs, /definitions/... gets a specific one.

Reply:
That kind of control has been done on the Router definition, on the ReydrathionProvider.

Indeed on that component we've splitted into 2 behaviour the /definitions  routes.
Using "exact" prop of react-router, we've assumed that when the route is exactly "/definitions" then it will show the Browse Page.
Otherwise, if the url contains some other data, it will show the FullDetailPage.

This make sense for you?

Reply:
It does make sense but `ROUTE_DETAIL` is not used anywhere that I can see. It is imported in the RehydrationProvider bu not used.